src/honeypot-stack.ts

```diff
@@ -60,26 +55,18 @@ export class HoneypotStack extends cdk.Stack {
 
     const secretsHelper = (task: string, key: string) => {
       const parameter = ssm.StringParameter.fromSecureStringParameterAttributes(this, `${task}${key}`, {
-        parameterName: `/all/${this.hostname}/${key}`,
+        parameterName: `/all/${props.hostnamePrefix}/${key}`,
```

Comment:
I'd probably just use stack name here since that will contain both the service name and the stage such as 'honeypot-prod' and 'honeypot-test'. If you accept this, you should also change the readme to match.
```suggestion
        parameterName: `/all/${this.stackName}/${key}`,
```